03_SageMakerPipelinesAndDeploy/docker/entrypoint.R

```diff
@@ -0,0 +1,45 @@
+library(jsonlite)
+library(reticulate)
+library(stringr)
+
+
+args = commandArgs(trailingOnly=TRUE)
+print(args)
+
+boto3 <- import('boto3')
+s3 <- boto3$client('s3')
+
+# Setup parameters
+# Container directories
+prefix <- '/opt/ml'
+input_path <- paste(prefix, 'input/data', sep='/')
+output_path <- paste(prefix, 'output', sep='/')
+model_path <- paste(prefix, 'model', sep='/')
+code_dir <- paste(prefix, 'code', sep='/')
+inference_code_dir <- paste(model_path, 'code', sep='/')
+
+# This is where the hyperparamters are saved by the estimator on the container instance
+param_path <- paste(prefix, 'input/config/hyperparameters.json', sep='/')
+
+# if param file exists then it is a training job, otherwise it's inference
```

Comment:
I believe the first arg to entrypoint would either be `serve` or `train`. Feels much safer to make this distinction based on that instead of a param file

Reply:
my bad - hadn't updated this, fixing now